Makefile

```diff
@@ -231,8 +225,15 @@ deploy_on_ocp_cluster: bring_assisted_installer
 		DEPLOY_TAG=$(DEPLOY_TAG) DEPLOY_MANIFEST_TAG=$(DEPLOY_MANIFEST_TAG) OCP_KUBECONFIG=$(OCP_KUBECONFIG) PROFILE=$(PROFILE) \
 		scripts/deploy_ui.sh
 
+deploy_controller_on_ocp_cluster: bring_assisted_installer
```

Comment:
Consider renaming it to day_2_controller

Reply:
Moved back to 'deploy_on_ocp_cluster' target, so already removed this one :)